src/map/utils/charutils.cpp

```diff
@@ -1683,7 +1683,7 @@ namespace charutils
         {
             auto removeSlotID = PItem->getRemoveSlotId();
 
-            for (auto i = 0; i < sizeof(removeSlotID) * 8; ++i)
+            for (auto i = 0; i < SLOT_FEET + 1; ++i)
```

Comment:
I'll take your second bit of advice on that, and since I don't know how to recommit this pull request with the changes I'll make a new pull request.